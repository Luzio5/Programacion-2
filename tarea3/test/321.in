# prueba de es_vacio_binario, insertar_en_binario, cantidad_binario, raiz, izquierdo, derecho, altura_binario, imprimir_binario, buscar_subarbol
es_vacio_binario
cantidad_binario
altura_binario
raiz
izquierdo
derecho
imprimir_binario
buscar_subarbol a
insertar_en_binario (10,l)
es_vacio_binario
cantidad_binario
altura_binario
raiz
izquierdo
derecho
imprimir_binario
buscar_subarbol a
buscar_subarbol l
insertar_en_binario (8,g)
insertar_en_binario (13,o)
es_vacio_binario
cantidad_binario
altura_binario
raiz
izquierdo
derecho
imprimir_binario
buscar_subarbol g
buscar_subarbol o
insertar_en_binario (9,h)
cantidad_binario
altura_binario
insertar_en_binario (6,e)
insertar_en_binario (4,b)
insertar_en_binario (12,n)
cantidad_binario
altura_binario
insertar_en_binario (15,q)
insertar_en_binario (7,f)
es_vacio_binario
cantidad_binario
altura_binario
raiz
izquierdo
derecho
imprimir_binario
buscar_subarbol e
buscar_subarbol f
Fin
