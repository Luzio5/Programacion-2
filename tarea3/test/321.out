1>#  prueba de es_vacio_binario, insertar_en_binario, cantidad_binario, raiz, izquierdo, derecho, altura_binario, imprimir_binario, buscar_subarbol.
2>Vacio.
3>0.
4>0.
5>b es vacío.
6>b es vacío.
7>b es vacío.
8>
9>sub es vacío.
10>Insertado.
11>No vacío.
12>1.
13>1.
14>(10,l)
15>
16>
17>
(10,l)
18>sub es vacío.
19>(10,l)
20>Insertado.
21>Insertado.
22>No vacío.
23>3.
24>2.
25>(10,l)
26>
(8,g)
27>
(13,o)
28>
-(13,o)
(10,l)
-(8,g)
29>(8,g)
30>(13,o)
31>Insertado.
32>4.
33>3.
34>Insertado.
35>Insertado.
36>Insertado.
37>7.
38>4.
39>Insertado.
40>Insertado.
41>No vacío.
42>9.
43>4.
44>(10,l)
45>
-(9,h)
(8,g)
--(7,f)
-(6,e)
--(4,b)
46>
-(15,q)
(13,o)
-(12,n)
47>
--(15,q)
-(13,o)
--(12,n)
(10,l)
--(9,h)
-(8,g)
---(7,f)
--(6,e)
---(4,b)
48>(6,e)
49>(7,f)
50>Fin.
